### DISCUSSION TITLE
Feat. Request: Daily commit reminder (think Duolingo)

### DISCUSSION THREAD

**Discussion by justin-gurtz** [community/community#86506] [Mobile] [answered]
GitHub already has a great commit visualization on the profile page but I'd also like daily push notifications when I haven't committed something yet, sort of like Duolingo when I haven't practiced yet. Might also be nice to include my current "streak" in that.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.143.0 (118103692)
Device: iPhone15,2 (iOS 17.2.1)
</details>

**Comment by stevepopovich** [comment]
Hi @justin-gurtz bit confused here, you just want a daily push notification if you have not already committed anything to any repo for that day? And gamify commits a bit more?

**Reply by justin-gurtz** [reply]
Exactly

I try to ship code everyday along with some other healthy habits (learning a language, exercise, etc) and occasionally by mid-afternoon I realize I've forgotten to code

**Reply by stevepopovich** [reply, marked as answer]
Gotcha, ok I will pass this on to the team for consideration
